src/components/cluster/detail/cluster_detail_view.js

```diff
@@ -61,7 +61,7 @@ class ClusterDetailView extends React.Component {
   registerRefreshInterval = () => {
     this.loadDataInterval = this.props.setInterval(
       this.loadDetails,
-      30 * 1000 // 30 seconds
+      3 * 1000 // 30 seconds
```

Comment:
Is this intended to stay at 3? I think it was probably for testing right

Reply:
Yeah, just for testing... thanks! 🤦‍♂